packages/tocco-ui/src/SearchBox/SearchBox.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import Button from '../Button'
+import './styles.scss'
+import _debounce from 'lodash/debounce'
+
+/**
+ * SearchBox
+ */
+class SearchBox extends React.Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      inputValue: ''
+    }
+    this.search = _debounce(this.search, props.debounce)
+  }
+
+  updateValue = evt => {
+    this.setState({'inputValue': evt.target.value})
+  }
+
+  search = () => {
```

Comment:
I discussed this behaviour also with [uwolfer]. I think this is the better approach. Otherwise it could be confusing. With this approach it is always possible to set the minInputLength to `1` or the debounce value to `0`. If I would implement your suggested approach, these two properties would have no effect.